### PR TITLE
Allow main CTA to be empty #165

### LINKED ISSUE #165: Allow announcement main CTA to be empty
On the announcement (showing at the top of the home screen), we should allow the main CTA (`"button_label"`) to potentially be empty (empty or `null`).

In this case, only `Dismiss` would show up in the announcement. 

**Why?** 
Sometimes we just want to give some info without necessarily have a main action to perform.

### PR DESCRIPTION
**Pull request by harshadmanglani**

### CODE REVIEW

File: lib/widgets/home/courses_row_item_widget.dart [hunk 1]
```diff
@@ -50,7 +50,7 @@ class CoursesRowItemWidget extends StatelessWidget {
     if (data != null) {
       return Text(data.subtitle, style: Theme.of(context).textTheme.subtitle1);
     } else {
-      return LoadingTextBoxWidget(height: 21);
+      return Container(height: 0, width: 0); //LoadingTextBoxWidget(height: 21);
```
Comment (michaelcspeed) [inline lib/widgets/home/courses_row_item_widget.dart:53]: What's the idea here @harshadmanglani ?

Comment (harshadmanglani) [reply lib/widgets/home/courses_row_item_widget.dart:53]: Oh, this was just removing the extra `LoadingTextBoxWidget` when the data loads. I sent you a screenshot on Slack, if you remember. Let me resend it.

File: lib/widgets/packs/announcement_banner_widget.dart [hunk 2]
```diff
@@ -78,19 +79,17 @@ class AnnouncementBannerState extends State<AnnouncementBanner>
...
+                                const EdgeInsets.only(top: 20.0, bottom: 10.0),
+                            child: MaterialBanner(
+                                backgroundColor: MeditoColors.darkMoon,
+                                content: _buildTextAndButtonColumn(
+                                    snapshot, context),
+                                leading:
+                                    snapshot.data.icon.isNotEmptyAndNotNull()
+                                        ? buildCircleAvatar(snapshot)
+                                        : Container(),
+                                actions: [Container()])),
```
Comment (michaelcspeed) [inline lib/widgets/packs/announcement_banner_widget.dart:91]: You don't wanna use this `actions` anymore?

File: lib/widgets/home/courses_row_item_widget.dart (hunk 1 above)
Comment (harshadmanglani) [reply lib/widgets/home/courses_row_item_widget.dart:53]: LoadingTextBox

Comment (harshadmanglani) [reply lib/widgets/home/courses_row_item_widget.dart:53]: Left image: original
Right image: changed

Feel free to undo that :) I was just trying something out

File: lib/widgets/packs/announcement_banner_widget.dart (hunk 2 above)
Comment (harshadmanglani) [reply lib/widgets/packs/announcement_banner_widget.dart:91]: The thing is, if I use `actions`, and when the main CTA is empty, it aligns the text and the dismiss button side by side.

Comment (harshadmanglani) [reply lib/widgets/packs/announcement_banner_widget.dart:91]: https://github.com/meditohq/medito-app/issues/165#issuecomment-913371196

**Comment by harshadmanglani** [comment]
❗❗❗❗ Also, before you merge, the Dismiss button isn't working. I can't figure out why.

File: lib/widgets/home/courses_row_item_widget.dart (hunk 1 above)
Comment (michaelcspeed) [reply lib/widgets/home/courses_row_item_widget.dart:53]: Ah yeah, looks much better now thanks :)

File: lib/widgets/packs/announcement_banner_widget.dart (hunk 2 above)
Comment (michaelcspeed) [reply lib/widgets/packs/announcement_banner_widget.dart:91]: Oh yeah super weird!

**Comment by michaelcspeed** [comment]
I will see if I can fix it :)

**Comment by michaelcspeed** [comment]
Fixed! Thanks again :) @harshadmanglani

**Pull request merged**